README.md

```diff
@@ -1,29 +1,31 @@
-# **Multilabel classification tutorial**
+# **Multilabel Classification with Metric Learning**
 
-This repository supports training models in classic miltilabel classification setup.
-It contains implementations of [MLDecoder](https://arxiv.org/abs/2111.12933),
-[ASL](https://arxiv.org/abs/2009.14119) and [AAM]() losses, a modified version of [ML-GCN](https://arxiv.org/abs/1904.03582).
+This repository contains code for the paper [K. Prokofiev, V. Sovrasov - Combining Metric Learning and Attention Heads For Accurate and Efficient Multilabel Image Classification](https://arxiv.org/abs/2209.06585).
+Also it contains the implementations of [MLDecoder](https://arxiv.org/abs/2111.12933),
+[ASL](https://arxiv.org/abs/2009.14119) and [AAM](https://arxiv.org/abs/2209.06585) losses, [TResNet-L](https://arxiv.org/abs/2003.13630) backbone and a modified version of [ML-GCN](https://arxiv.org/abs/1904.03582).
 Supported backbones table:
 
 Model         | Input Resolution | MACs(G)* | mAP on COCO 14* | mAP on PASCAL VOC 07*
 ---           |---               |---        |---      |---
 EfficientNetV2-S ([OI weights](https://drive.google.com/uc?export=download&id=1N0t0eShJS3L1cDiY8HTweKfPKJ55h141) )       |448x448           | 12.28    | 88.75       | 95.86
 EfficientNetV2-L                 |448x448     | 49.92    | 90.10       | 96.05
 TResNet-L                        |448x448     | 36.15    | 90.30       | 96.70
+
 *with ML-Decoder and AAM loss
 
 ## Data format
 Annotation for multilabel classification is stored in json format.
-Root of the json file contatins a list of classes available by key `classes` and
-a lsit of images with labels available by key `images`. Annotation of a single
+Root of the json file contains a list of classes available by key `classes` and
+a list of images with labels available by key `images`. Annotation of a single
 image is a tuple containing a relative path to the image as a first element and a list of all
 the presented labels as a second element.
 
 Annotations and [download instructions](./datasets/README.md) for COCO14, NUS WIDE, VG500 and VOC07 can be found in the `datasets` folder.
 
 ## Environment setup
 
-In a clean python environment install an appropriate for your GPU configuraton build of pytorch 1.8.2 LTS.
+In a clean python environment install an appropriate for your GPU configuration build
+of [pytorch](https://pytorch.org/get-started/locally/) 1.8.2 LTS.
```

Comment:
bash init_venv.sh, no?

Reply:
This script works only when d-o-r is a submodule of OTE

Reply:
Ok